tests/typedefs.py

```diff
@@ -0,0 +1,6 @@
+from aiohttp import web
+from aiohttp.test_utils import TestClient
+
+from typing import Awaitable, Callable
+
+_TAiohttpClient = Callable[[web.Application], Awaitable[TestClient]]
```

Comment:
Isn't this for `aiohttp.pytest_plugin`? Would probably be better if we included the type in `aiohttp` directly and then import it into other projects, rather than needing to redefine it in every project.

Reply:
> Isn't this for aiohttp.pytest_plugin?

Yes.

> Would probably be better if we included the type in aiohttp directly and then import it into other projects, rather than needing to redefine it in every project.

True, but given the current wait for `aiohttp 3.8` I would opt to add it now, with a note to replace it when `aiohttp` provides typing for that. (will also check whether typing for this is already provided in `aiohttp`, since it has been a while :D )

Reply:
I couldn't see anything, but feel free to make an extra PR to add it.

Agreed that we'll need this here as well for now.